workers/unity/Packages/com.improbable.gdk.transformsynchronization/Systems/RateLimitedTransformSendSystem.cs

```diff
@@ -45,18 +45,17 @@ protected override void OnUpdate()
 
                 var transformToSend = data.TransformToSend[i];
 
-                var currentTransform = new TransformInternal.Component
-                {
-                    Location = (transformToSend.Position - worker.Origin).ToImprobableLocation(),
-                    Rotation = transformToSend.Orientation.ToImprobableQuaternion(),
-                    Velocity = transformToSend.Velocity.ToImprobableVelocity(),
-                    PhysicsTick = transform.PhysicsTick + data.TicksSinceLastUpdate[i].NumberOfTicks,
-                    TicksPerSecond = tickRate.PhysicsTicksPerRealSecond,
-                    DirtyBit = true
-                };
+                var currentTransform = new TransformInternal.Component();
```

Comment:
Why the change away from object initializer syntax?

Reply:
Quite, but object initialization syntax doesn't bypass the `set` methods, does it?

Reply:
Uhh, that very nice. Changed back to old syntax.